### ISSUE TITLE
Anti-DDoS Infrastructure: Change mitigation profile of an IP address

### ISSUE THREAD

**Issue by VibeGAMESNL** [ovh/infrastructure-roadmap#116]
As an user, I know OVH support can change the mitigation profile for an IP address. I want to be able to change a mitigation profile or certain pre-defined rate-limits for each of my IP addresses myself.

It happends a lot were an attack causes the VAC to drop legit traffic because it reaches a certain limit. I want more flexibility and possibly fix it myself before I have to contact the support. Something similar is already implemented for the GAME server line-up.

Additionally it would be nice if I can see which profile is active for which IP in my manager, even if the support changed the profile or its limits.

**Comment by ytendx** [comment]
Hey guys,
Would highly appreciate this to be done.

**Comment by AbdulOmarCR** [comment]
Yeah I'd like to be able to manage my IPs profiles

**Comment by tranceuser** [comment]
I'd like to ask if there's an update.

**Comment by gasanache** [comment]
Bump.

**Comment by VibeGAMESNL** [comment]
Knowning OVH, probably never gonna happen

**Comment by VibeGAMESNL** [comment]
> Bump.

Knowing OVH, probably never gonna happen. They only focus on bulk and don't care about the few that want a little more control. We ended up migrating 150 servers from OVH 2 years ago because they simply refuse to implement important features. Dont waste your time here.

**Comment by jslocinski** [comment]
We're preparing related bricks to be able to offer this feature. 

Acknowledged items are aligned with our evolution strategy, but not yet prioritized. Once we're sure about prioritization within 12m, the feature will be moved to Prioritized accordingly.

**Comment by Stereo973** [comment]
Would love it !

**Comment by gasanache** [comment]
Any news?

**Comment by mrc4tt** [comment]
@tranceuser, probably 2026-2027.

**Comment by jslocinski** [comment]
For now it works in semi-manual mode.
Most recent profile available via Support request is Blockchain/Solana. For that please share Solana nodes' IPs to our Support teams mentioning to add them into Anti-DDoS Solana profile.

**Comment by gasanache** [comment]
> For now it works in semi-manual mode. Most recent profile available via Support request is Blockchain/Solana. For that please share Solana nodes' IPs to our Support teams mentioning to add them into Anti-DDoS Solana profile.

We're more interested in tuning one for a Dedicated Server.

**Comment by jslocinski** [comment]
As mentioned - please reach out to our support teams for that.

**Comment by bigfrankvis-gif** [comment]
> As mentioned - please reach out to our support teams for that.

And wait 4 weeks lol

**Comment by VoNhatWang** [comment]
POV: You are subjected to continuous DDoS attacks for hours, even days, but have to wait a whole day for OVH to respond to one message, not to mention times when you provide incomplete information or OVH misprocesses your information.

This feature is extremely necessary. It should be available for the Advance Server as well, instead of just the Game Server.

**Comment by jslocinski** [comment]
We've just opened an Alpha phase of Anti-DDoS profiling. You can request participation by filling-in the survey, we will contact first testers to onboard soon!

https://labs.ovhcloud.com/en/antiddos-profiles/

**Comment by Gawnz1** [comment]
@jslocinski 

```
Q4. Which profiles are available during the alpha?
We prioritize VPN/UDP, Blockchain/Solana and Streaming protocols first. Additional profiles are planned for the later evolution.
```

Please review https://github.com/ovh/infrastructure-roadmap/issues/177 as priority profile for next or some tunning options.

**Comment by jslocinski** [comment]
> [@jslocinski](https://github.com/jslocinski)
> 
> ```
> Q4. Which profiles are available during the alpha?
> We prioritize VPN/UDP, Blockchain/Solana and Streaming protocols first. Additional profiles are planned for the later evolution.
> ```
> 
> Please review [#177](https://github.com/ovh/infrastructure-roadmap/issues/177) as priority profile for next or some tunning options.

Yes, we have this in mind, but it's separate topic and let's not mix them for clarity.

**Comment by amsaal** [comment]
I have submiited the survey. I had issues on TeamSpeak 3 which was crashing/timeout due to impact the mitigation was taking place.   I am on Gen3 firewall which is like newest ddos protection.

**Comment by jslocinski** [comment]
> I have submiited the survey. I had issues on TeamSpeak 3 which was crashing/timeout due to impact the mitigation was taking place. I am on Gen3 firewall which is like newest ddos protection.

Thanks, we will start contacting first onboarded testers soon. 

PS. Please note that IP addresses pointing GAME servers are for now excluded. Main objective for profiling Anti-DDoS are in fact any other applications (since GAME is Layer 7 protection and will get it's own advanced filtering in coming months)
